### PR TITLE
view cart summary

### PR DESCRIPTION
**Pull request by JeffNewcomer**
jamazon 5

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -276,25 +283,130 @@ function displayLargeCard() {
...
+function renderCartPage(cartObjects) {
+  var $cartHeader = document.createElement('h1')
+  var $cartListContainer = document.createElement('div')
+  $cartListContainer.classList.add('cartListContainer')
+  $cartHeader.textContent = 'Cart'
+  $cartHeader.classList.add('headerText')
+  $cartListContainer.appendChild($cartHeader)
+
+  for (var k = 0; k < cartObjects.length; k++) {
+    $cartListContainer.appendChild(createCartItem(app.cart.items[k]))
```
Comment (thebearingedge) [inline main.js]: Your `cartObjects` parameter should contain the items. No need to reach out to `app.cart.items`.

File: main.js [hunk 2]
```diff
@@ -276,25 +283,130 @@ function displayLargeCard() {
...
+  $continueShoppingButton.classList.add('btn', 'btn-primary')
+  $continueShoppingButton.textContent = 'Continue Shopping'
+
+  $continueShoppingButton.addEventListener('click', returnToCatalog)
+
+  return $cartListContainer
+}
+
+function returnToCatalog() {
+  $catalog.innerHTML = ''
```
Comment (thebearingedge) [inline main.js]: You shouldn't need to empty the views in this function. Your `renderAppState` function should be smart enough to empty out the different views when appropriate.

File: styles.css [hunk 3]
```diff
@@ -4,7 +4,7 @@ body {
   font-family: 'Raleway', sans-serif;
 }
 
-.jamazonText {
+.headerText {
```
Comment (thebearingedge) [inline styles.css]: Stick with `kebab-case` convention for your CSS class names and ids.

File: main.js [hunk 4]
```diff
@@ -276,25 +283,130 @@ function displayLargeCard() {
...
+$cartCounter.addEventListener('click', openTheCart)
+
+function openTheCart() {
+  $catalog.innerHTML = ''
+  $details.innerHTML = ''
+  app.view = 'cart'
+  renderAppState()
+}
+
+function createCartItem(item) { // change to (cartItem)?
```
Comment (thebearingedge) [inline main.js]: You should remove little comments like this.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Alright! A couple small changes.

File: main.js (hunk 1 above)
Comment (JeffNewcomer) [reply main.js]: screen shot 2018-07-05 at 1 10 16 pm

File: main.js (hunk 2 above)
Comment (JeffNewcomer) [reply main.js]: screen shot 2018-07-05 at 1 10 58 pm
screen shot 2018-07-05 at 1 11 39 pm

File: styles.css (hunk 3 above)
Comment (JeffNewcomer) [reply styles.css]: screen shot 2018-07-05 at 1 12 38 pm

File: main.js (hunk 4 above)
Comment (JeffNewcomer) [reply main.js]: done
screen shot 2018-07-05 at 1 13 51 pm

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**